### ISSUE TITLE
"Required" icon appears twice

### ISSUE THREAD

**Issue by OhioGuy67** [formvalidation/formvalidation#1349]
In my application, I had combined the "Showing required icon for the mandatory fields" and "Showing icons in custom area" examples and it was working great (except the status icons were appearing after the fields and not after the label, but that was ok because I liked having them appear after the field).  I added the code from the "Adding dynamic field" example.  The problems that are occurring is the required icon is appearing twice in the initial field (it appears only once for a new field that is added) and if the field fails validation, the icon on the initial field is not being changed to the X (the field is being highlighted in red and the error message displayed correctly).  I've traced some of the problem down to the $icon.insertAfter($label); line in init.field.fv.  If I remove this line, everything works correctly except the icon appears under the field.  So I'm guessing that I've done something silly.  Can you please tell me what I'm doing wrong?  Unfortunately I cannot provide my code since it's on a standalone, isolated system so I can't get it off (and have to hand type in everything).

**Comment by nghuuphuoc** [comment]
> Unfortunately I cannot provide my code

Please, try to give me the sample form demonstrating your case. Without this, it's not easy for me to figure it out.

**Comment by OhioGuy67** [comment]
Thank you for replying to my issue.  I was actually able to get things working yesterday (don't remember what exactly I did) and hadn't gotten around to mentioning that here.

Thanks again.

**Comment by OhioGuy67** [comment]
Thank you for replying to my issue.  I was actually able to get things working yesterday by tweaking the cloning process so that the "template" field in the HTML did not have to be a duplicate of the "original" field.  This took care of the problem and I hadn't gotten a chance to post here that things are now working.

Thanks again.
